app/actions/action_dropNewPin.js

```diff
@@ -1,5 +1,5 @@
-import { DROP_NEW_PIN } from '../constants/constants.js';
-import { userData } from '../lib/db/db.js';
+import { DROP_NEW_PIN, UPDATE_RECENT, SET_RECENT} from '../constants/constants.js';
```

Comment:
We should consider renaming this file since it now has several action creators and ultimately exports `getsLocationToSave`.